### PR TITLE
Mobility GHC927 rebased packet-set upgrade

### PR DESCRIPTION
**Pull request by arjunkathuria**
supersedes #6 

This PR contains GHC-9.2.7 package set updates (same as #5 , #8), rebased for updated haskell-flake changes, minus the ormolu changes in #10

### CODE REVIEW

**Comment by arjunkathuria** [comment]
^note:- rebased again - 16/06/23

rebased again - newest nammayatri commits failed to build without newer changes in the common repo.

**Comment by srid** [comment]
@arjunkathuria When will GHC 9 support be merged to `main` in ny repos? What's left to do?

**Review by srid — APPROVED** [review]
LGTM.

Merge this in sync with the other PRs, once they too are approved.

**Comment by srid** [comment]
Closing since https://github.com/nammayatri/common/pull/18 was merged.

**Pull request closed without merging**
